scripts/build-plugins/rollup-plugin-build-themes.js

```diff
@@ -254,13 +254,50 @@ module.exports = function buildThemes(options) {
                 const derivedVariables = compiledVariables["derived-variables"];
                 const icon = compiledVariables["icon"];
                 const builtAssets = {};
-                /**
-                 * Generate a mapping from theme name to asset hashed location of said theme in build output.
-                 * This can be used to enumerate themes during runtime.
-                 */
+                let defaultDarkVariant = {}, defaultLightVariant = {};
```

Comment:
I know this is from a previous PR, but I'm struggling a bit to read this code because it's not clear what assetMap & chunkMap contain, and what parseBundle does specifically. Perhaps some more detailed naming and/or comments could help here.